db/seeds.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+ActionMailer::Base.perform_deliveries = false
```

Comment:
Do we want to only do this if it is the development environment? 
@katelovescode -- do you use db:seed with staging, for example?

Reply:
I don't seed staging, no.  I think we're okay to do this no matter what.  If we end up seeding and/or re-seeding staging, we probably don't want emails to go out anyway.